File: dongyimai_manager_web/src/main/webapp/js/service/contentService.js
//服务层
app.service('contentService',function($http){
	    	
	//findAll
	this.findAll=function(){
		return $http.get('../content/findAll.do');		
	}

	//findPage
	this.findPage=function(page,rows){
		return $http.get('../content/findPage.do?page='+page+'&rows='+rows);
	}

	//findOne
	this.findOne=function(id){
		return $http.get('../content/findOne.do?id='+id);
	}

	//add
	this.add=function(entity){
		return  $http.post('../content/add.do',entity );
	}

	//update
	this.update=function(entity){
		return  $http.post('../content/update.do',entity );
	}

	//dele
	this.dele=function(ids){
		return $http.get('../content/delete.do?ids='+ids);
	}

	//条件、模糊查询+分页
	this.search=function(page,rows,searchEntity){
		return $http.post('../content/search.do?page='+page+"&rows="+rows, searchEntity);
	}

});

File: dongyimai_content_interface/src/main/java/com/offcn/content/service/ContentService.java
package com.offcn.content.service;

import com.offcn.entity.PageResult;
import com.offcn.pojo.TbContent;

import java.util.List;

public interface ContentService {

	//findAll
	public List<TbContent> findAll();
	
	//findPage
	public PageResult findPage(int pageNum, int pageSize);
	
	//add
	public void add(TbContent content);
	
	//updata
	public void update(TbContent content);
	
	//findOne
	public TbContent findOne(Long id);
	
	//delete
	public void delete(Long[] ids);

	//条件、模糊查询+分页
	public PageResult findPage(TbContent content, int pageNum, int pageSize);

	/**
	 * 通过广告分类ID查询本类广告列表（网站前台→首页广告轮播）
	 * @param categoryId
	 * @return
	 */
	public List<TbContent> findContentByCategoryId(Long categoryId);
	
}
